src/org/gk/render/AbstractNodeRenderer.java

```diff
@@ -51,32 +52,32 @@ public void render(Graphics g) {
     }
 
     protected abstract void renderShapes(Graphics g);
-    
-    protected void renderDrug(Graphics g) {
+
+    protected void renderDrugLabel(Graphics g) {
         // Want to use a smaller font
         Font font = g.getFont();
         Font oldFond = font;
         font = font.deriveFont(Font.ITALIC + Font.BOLD, font.getSize() - 2);
         Graphics2D g2 = (Graphics2D) g;
         Rectangle2D textBounds = font.getStringBounds(DRUG_SYMBOL, g2.getFontRenderContext());
         // Make sure it is drawn at the bottom right corner
-        Rectangle bounds = node.getBounds();
+        Rectangle labelBounds = node.getBounds();
+
+        // Width and Height of the label.
         int w = (int)(textBounds.getWidth() + boundsBuffer);
         int h = (int)(textBounds.getHeight() + boundsBuffer);
-        int x = (int) (bounds.getMaxX() - w);
-        int y = (int) (bounds.getMaxY() - h);
-        
+        Dimension labelDimensions = new Dimension(w, h);
+
+        // Bottom right coordinates.
+        int x = (int)(labelBounds.getMaxX() - labelDimensions.getWidth());
+        int y = (int)(labelBounds.getMaxY() - labelDimensions.getHeight());
         // Some shapes for the labeling
-        if (background == null) { 
+        if (background == null) {
             g2.setPaint(DEFAULT_BACKGROUND);
         }
         else {
             g2.setPaint(background);
         }
-        g2.fillRect(x, y, w, h);
-        setDrawPaintAndStroke(g2);
-        g2.drawRect(x, y, w, h);
-        
         // Draw the drug label
```

Comment:
Please make sure these three lines are not needed really.

Reply:
I removed these three liens to more closely match the browser pathways, but can absolutely add them again!

With these three lines:
<img width="172" alt="Screen Shot 2019-12-20 at 10 06 58 AM" src="https://user-images.githubusercontent.com/15309567/71280601-81104f00-2310-11ea-887c-3ca0bc568e66.png">

Without these three lines:
<img width="172" alt="Screen Shot 2019-12-20 at 10 07 09 AM" src="https://user-images.githubusercontent.com/15309567/71280602-81a8e580-2310-11ea-8b20-5a9c1a0152ef.png">